traffic_server/_tsb/run.sh

```diff
@@ -73,4 +73,4 @@ ED
 # This includes changing output redirection to traffic.out and adding udev-settle to wait for disks
 (sed -i 's/ExecStart=@exp_bindir@\/traffic_manager \$TM_DAEMON_ARGS/ExecStart=@exp_bindir@\/traffic_manager --bind_stdout @exp_logdir@\/traffic.out --bind_stderr @exp_logdir@\/traffic.out \$TM_DAEMON_ARGS/g' /rpmbuilddir/SOURCES/src/rc/trafficserver.service.in)
 (sed -i 's/After=syslog.target network.target/Wants=systemd-udev-settle.service \nAfter=syslog.target network.target systemd-udev-settle.service/g' /rpmbuilddir/SOURCES/src/rc/trafficserver.service.in)
-rpmbuild -bb ${rpmbuild_openssl} --define "_topdir /rpmbuilddir" /rpmbuilddir/SPECS/trafficserver.spec || die "Failed to build rpm."
+rpmbuild -bb ${rpmbuild_openssl} --define "_topdir /rpmbuilddir" --define "ats_version 9.2.3" /rpmbuilddir/SPECS/trafficserver.spec || die "Failed to build rpm."
```

Comment:
Reuse %tag instead of hard-coding `9.2.3`?

https://github.com/apache/trafficcontrol/blob/3cd6460cc39afd10b12a01babf12f849d63ceb4f/traffic_server/_tsb/trafficserver.spec#L23

Reply:
updated @zrhoffman